organizations/tests/test_admin.py

```diff
@@ -105,9 +107,13 @@ def test_foreign_key_field_active_choices(self):
         Test: organization course foreignkey widget has active organization choices.
         """
         create_organization(active=True)
+        if sys.version_info < (3, 0):
+            choice = 'test organization ()'
+        else:
+            choice = 'Organization object'
```

Comment:
You shouldn't need this if you use the [python_2_unicode_compatible](https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.encoding.python_2_unicode_compatible) decorator on the `Organization` model.

Reply:
Great - done!